src/price/service.price.platform.hpp

```diff
@@ -31,16 +31,28 @@ namespace antara::mmbot
 {
     using registry_price_result = std::unordered_map<antara::pair, st_price>;
 
-    class price_service_platform
+    class abstract_price_service_platform
```

Comment:
I really disagree as using abstract class for price_service_platform (there is only one service)

Use template class instead